components/analytics/system_analytics/format.jsx

```diff
@@ -0,0 +1,85 @@
+import * as Utils from 'utils/utils.jsx';
+
+export function formatChannelDoughtnutData(totalPublic, totalPrivate) {
```

Comment:
This isn't new code, just refactored out of the `system_analytics` component for clarity.